Dockerfile.alpine-go

```diff
@@ -0,0 +1,9 @@
+FROM golang:1.14-alpine3.12
+
+RUN set -eux; \
+    apk add --no-cache \
+    gcc \
```

Comment:
This is included in build-base: https://pkgs.alpinelinux.org/package/v3.12/main/x86_64/build-base